python/src/aiconfig/default_parsers/azure.py

```diff
@@ -0,0 +1,40 @@
+from aiconfig.default_parsers.openai import DefaultOpenAIParser
+from aiconfig.util.config_utils import get_api_key_from_environment
+
+from openai import AzureOpenAI
+
+
+class AzureParser(DefaultOpenAIParser):
+    def __init__(self, deployment: str):
```

Comment:
Alternatively this can take a mapping of deployment and model name, but I think this is better the way you've done it